pennylane_forest/qpu.py

```diff
@@ -106,6 +98,17 @@ def __init__(
         """Union[None, pyquil.ExecutableDesignator]: the latest compiled program. If parametric
         compilation is turned on, this will be a parametric program."""
 
+
+        if "parametric_compilation" in kwargs and kwargs["parametric_compilation"]:
```

Comment:
This can probably be simplified to
```suggestion
        if kwargs.get("parametric_compilation", False):
```

Reply:
(Is this False by default?)

Reply:
It's not included in the signature, later defaults to `True`. Here we would like to check for the case when the user has explicitly turned on parametric compilation